terraform/aws-accounts/cloud-platform-aws/account/opensearch.tf

```diff
@@ -81,6 +81,30 @@ resource "aws_kms_key" "live_modsec_audit" {
   multi_region                       = false
 }
 
+data "aws_route53_zone" "et_cloud_platform_justice_gov_uk" {
```

Comment:
Just a cosmetic change. Can we name the resource as `cloud_platform_justice_gov_uk`

Reply:
good catch